testsuite/integration/expansion/src/test/java/org/wildfly/test/integration/observability/opentelemetry/OpenTelemetryMultipleWarTestCase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.opentelemetry;
+
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Objects;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.Response;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.InSequence;
+import org.jboss.arquillian.testcontainers.api.DockerRequired;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.shared.observability.setuptasks.OpenTelemetryWithCollectorSetupTask;
+import org.jboss.as.test.shared.observability.signals.PrometheusMetric;
+import org.jboss.as.test.shared.observability.signals.jaeger.JaegerSpan;
+import org.jboss.as.test.shared.observability.signals.jaeger.JaegerTrace;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.wildfly.test.integration.observability.opentelemetry.application.OtelMetricResource;
+
+/**
+ * This test verifies that two OpenTelemetry applications can be deployed to the same server while their
+ * observability signals remain distinct. It deploys two applications, and makes several requests to each to
+ * generate data. It then verifies that traces for each application are successfully exported to the Collector.
+ * Each trace will have a span with a tag of <code>url.path</code>. The value of that tag should be the name of
+ * the archive (i.e., 'service-one' or 'service-two'). Similarly, there should be a counter exported for each
+ * service with the tag <code>job</code> that contains the name of the archive. The value of the metric should
+ * equal <code>REQUEST_COUNT</code>.
+ */
+@RunAsClient
+@ServerSetup({OpenTelemetryWithCollectorSetupTask.class})
+@DockerRequired
+public class OpenTelemetryMultipleWarTestCase extends BaseOpenTelemetryTest {
+    protected static final String SERVICE_ONE = "service-one";
+    protected static final String SERVICE_TWO = "service-two";
+    private static final int REQUEST_COUNT = 1;
```

Comment:
This should have been higher (say, 5), but I dropped it to one so I had less debug output to sift through. I'll update this to something higher once I see what CI thinks of the changes.